libpod/runtime.go

```diff
@@ -25,6 +25,7 @@ type Runtime struct {
 	storageService *storageService
 	imageContext   *types.SystemContext
 	ociRuntime     *OCIRuntime
+	locksDir       string
```

Comment:
nit question.  You've lock and lockpath, but locksdir (plural).  I'd make this lockDir, but not a biggy.

Reply:
Yes lets do lockdir. If there is a difference between lockdir and lockpath.  Otherwise lockpath.